eslint-config/common-ts.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
   rules: {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-namespace': 'off',
+    'no-console': ['error', { allow: ['warn'] }],
```

Comment:
> You can now run next lint every time you want to run ESLint to catch errors. Once ESLint has been set up, it will also automatically run during every build (next build). Errors will fail the build, while warnings will not.

이거 바로 error로 셋팅하면 빌드 실패할 수도 있을것 같습니다. warn으로 적응기를 갖고 코드에 남아있는 console 코드들 제거한 다음에 error로 하는 것은 어떨까요?